packages/core/src/interpreter.ts

```diff
@@ -872,6 +872,13 @@ export class Interpreter<
         } else {
           if (sendAction.to) {
             this.sendTo(sendAction._event, sendAction.to);
+          } else if (sendAction.sendType === 'forward') {
```

Comment:
However, when we're _forwarding_ an event, it should never be forwarded to self! So we need a way of identifying that the send action is a "forwarding" action, without breaking any behavior (hence `sendAction.sendType === 'forward'`)